styles.css

```diff
@@ -334,3 +342,298 @@ footer h6 {
+    padding: 1rem;
+    margin: 0 auto;
+  }
+
+  .skils-container .cont {
+    display: inline;
+    width: 25%;
+    font-size: 18px;
+    font-weight: 700;
+  }
+
+  .list {
+    font-size: 14.67px;
+    font-weight: 400;
+    line-height: 28px;
+  }
+
+  .social { display: none; }
+
+  .work { display: none; }
+
+  .so-big {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    height: 2.5rem;
+    margin-top: 6rem;
+  }
+
+  .social-big {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 40%;
+    padding: 1rem;
+  }
+
+  .social-big li {
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  .work-titl {
+    font-size: 64px;
+    font-weight: 700;
+    margin-left: 5rem;
+  }
+
+  .work-titl-form {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .hr-big { width: 85%; }
+
+  .hr-last {
+    width: 100%;
+    height: 1px;
+    margin-top: 2rem;
+  }
+
+  .second-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    align-content: space-around;
+    background: url("img/1.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-top: 5rem;
+    margin-bottom: 7rem;
+  }
+
+  .second-container .card-container {
+    display: flex;
+    width: 100%;
+    grid-column: 1 / 3;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 2rem;
+  }
+
+  .card-1,
+  .card-3 {
+    width: 40%;
+    padding-top: 8px;
+  }
+
+  .card-2,
+  .card-4 {
+    width: 40%;
+    padding-top: 8px;
+    margin-left: 8rem;
+    margin-right: -7rem;
+  }
+
+  .card-1 p,
+  .card-2 p,
+  .card-3 p,
+  .card-4 p {
+    font-size: 20px;
+    font-weight: 400;
+  }
+
+  .con-1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 2rem;
+  }
+
+  .con {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .con-1 img,
+  .con img {
+    width: 60%;
+    margin-left: 8rem;
+  }
+
+  .list-container ul {
+    display: flex;
+    justify-content: space-around;
+    color: #d9d9ec;
+    padding-left: 0;
+    margin-bottom: 1rem;
+    width: 75%;
+  }
+
+  .hp-last {
+    width: 50%;
+    margin: 0 auto;
+    text-align: center;
+  }
+
+  .hp-last p { width: 100%; }
+
+  form {
+    display: block;
+    margin: 0 auto;
+    width: 80%;
+    height: fit-content;
+    text-align: center;
+    margin-top: 3rem;
+    margin-bottom: 1rem;
+  }
+
+  #user_name,
+  #user_mail {
+    background-color: #fff;
+    width: 55%;
+    height: 40px;
+    margin-bottom: 1.5rem;
+  }
+
+  #user_text {
+    width: 55%;
+    height: 148px;
+    background-color: #fff;
+    margin-bottom: 4rem;
+  }
+
+  form .btn { width: 20%; }
+
+  .social-last {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-left: 2rem;
+  }
```

Comment:
- [ ] [OPTIONAL] please add the styles based on user interaction on all your buttons. Like when `hovered`, `pressed`, `active`, etc. This enhances user interaction and user experience. They have provided a guide for those states in Figma 👍